app/src/docs/content/streams/usingStreamsInCore.mdx

```diff
@@ -32,10 +36,32 @@ Head over to the <Link to={routes.streams.index()}>Streams page</Link> to see an
   alt="Stream Listing"
 />
 
-This stream listing page gives an indication for when data was last pushed to each stream and uses traffic light indicators show the health of each stream, at a glance. Green shows a stream receiving data, while red indicates no data flowing. Clicking on a stream will open the stream editor. You may only edit streams that you have created or have explicit permission to edit.
+This stream listing page displays all streams the current user has a right to publish or subscribe to. Clicking on a stream will open the stream editor. You may only edit streams that you have created or have explicit permission to edit.
 
 By clicking on the more options icon you may access advanced features for each stream, such as stream sharing and code snippets. A new stream can be created by the clicking the Create Stream button.
 
+### Creating streams
+Streams are identified by a stream ID, which is an Ethereum address plus a user-given path, just like an Internet URL. The Ethereum address is referred to as the domain and is either the Ethereum address of the logged in user, or it can also be the ENS (Ethereum Name Service) name owned by the Ethereum address of the logged in user. All streams require a unique path in the format of domain/pathname and can contain single or multi-level paths.
```

Comment:
```suggestion
Streams are identified by a stream ID, which consists of a domain and a user-given path, just like an Internet URL. The domain is either an Ethereum address of the logged in user, or it can also be an ENS (Ethereum Name Service) name owned by the Ethereum address of the logged in user. All streams require a unique path in the format of `domain/pathname` and can contain single or multi-level paths.
```